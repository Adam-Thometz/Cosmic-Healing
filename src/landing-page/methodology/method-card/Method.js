import React from 'react';

import './Method.css';

const Method = ({ name, img, description }) => {
  return (
    <figure className='Method'>
      <img src={img} alt={name} />
      <figcaption>
        <h3>{name}</h3>
        <p>{description}</p>
      </figcaption>
    </figure>
  )
}

export default Method