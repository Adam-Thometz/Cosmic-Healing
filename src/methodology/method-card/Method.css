.Method {
  width: 25%;
}

.Method h3 {
  font-family: "Nimbus Roman";
  font-size: 24px;
}

.Method p {
  font-family: Nuckle;
  font-size: 18px;
}