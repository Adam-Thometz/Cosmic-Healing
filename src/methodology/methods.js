import dreamwork from '../_media/method-icons/dreamwork.png';
import shadowWork from '../_media/method-icons/shadow-work.png';
import emotionalProcessing from '../_media/method-icons/emotional-processing.png';
import motivationalInterviewing from '../_media/method-icons/motivational-interviewing.png';
import cognitiveBehavioral from '../_media/method-icons/cognitive-behavioral.png';
import personCentered from '../_media/method-icons/person-centered.png';

const methods = [
  {
    name: "Dreamwork",
    img: dreamwork,
    description: "Decode symbols within your  dreams to uncover subconscious messages"
  },
  {
    name: "Shadow Work",
    img: shadowWork,
    description: "Integrate the repressed and hidden parts of you"
  },
  {
    name: "Emotional Processing",
    img: emotionalProcessing,
    description: "Understand and process your emotions from the past and present"
  },
  {
    name: "Motivational Interviewing",
    img: motivationalInterviewing,
    description: "Find internal motivation to change self-destructive behavior"
  },
  {
    name: "Cognitive Behavioral",
    img: cognitiveBehavioral,
    description: "Release negative patterns and limiting beliefs"
  },
  {
    name: "Person Centered",
    img: personCentered,
    description: "Gain the confidence to take charge of your own life"
  },
];

export default methods