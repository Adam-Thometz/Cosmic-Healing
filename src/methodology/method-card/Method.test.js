import React from "react";
import { render } from "@testing-library/react";
import Method from './Method';

import methods from '../methods';

describe('Method component', () => {
  const { name, img, description } = methods[0]
  it('renders without crashing', () => {
    render(<Method
      name={name}
      img={img}
      description={description}
    />);
  });

  it('matches the snapshot', () => {
    const { asFragment } = render(<Method
      name={name}
      img={img}
      description={description}
    />);
    expect(asFragment()).toMatchSnapshot();
  });
});