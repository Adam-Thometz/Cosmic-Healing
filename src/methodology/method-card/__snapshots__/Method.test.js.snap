// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Method component matches the snapshot 1`] = `
<DocumentFragment>
  <figure
    class="Method"
  >
    <img
      alt="Dreamwork"
      src="dreamwork.png"
    />
    <figcaption>
      <h3>
        Dreamwork
      </h3>
      <p>
        Decode symbols within your dreams to uncover subconscious messages
      </p>
    </figcaption>
  </figure>
</DocumentFragment>
`;
